src/components/Select/Select.tsx

```diff
@@ -65,9 +72,12 @@ const Select = React.forwardRef<HTMLInputElement, Props>(
           size={20}
           name={open ? 'chevronLargeUp' : 'chevronLargeDown'}
           color={theme.utils.getColor('lightGray', 500)}
+          onClick={() => {
+            inputRef?.current?.focus();
```

Comment:
I think it is more UX-friendly to toggle only the list of options instead of focusing when you click on the arrow icon. 
This way you can also avoid the complexity of multiple refs.

example:
```
setOpen(!open);
```